ansible_collections/arista/avd/molecule/eos_designs-twodc-5stage-clos/inventory/group_vars/DC1_POD1.yml

```
@@ -108,7 +108,7 @@ l3leaf:
             domain_list:
               - structured-config.set.on.node
           # id: 2 Using pool manager for ID
-          # mgmt_ip: 192.168.1.8/16 Test without management IP
+          mgmt_ip: 192.168.1.8/16
```

Adding the mgmt_ip we loose the test coverage that was here for not having the mgmt_ip in combination with use_cv_topology and other things. It seems like we already had a behavior of silently ignoring peer keepalive if there was no mgmt_ip. That was not clear to me when you asked, so based on this, we should not change that behavior.

---

Before enabling mgmt_ip,
mlag configuration on leaf2a
```
mlag configuration
   domain-id RACK2_MLAG
   local-interface Vlan4094
   peer-address 172.20.110.3
   peer-address heartbeat 192.168.1.9 vrf MGMT
   peer-link Port-Channel5
   dual-primary detection delay 5 action errdisable all-interfaces
   reload-delay mlag 300
   reload-delay non-mlag 330
```
mlag configuration on leaf2b
```
mlag configuration
   domain-id RACK2_MLAG
   local-interface Vlan4094
   peer-address 172.20.110.2
   peer-link Port-Channel5
   reload-delay mlag 300
   reload-delay non-mlag 330
```
So one leaf was having dual primary detection configuration and other didnt, which created uneven mlag configuration.

---

understood, but we don't currently guard things like this between the mlag peers. I don't think we need to add itas part of dual primary detection.

---

Reverted the changes.